src/SlmLocale/Strategy/AssetStrategy.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace SlmLocale\Strategy;
+
+use SlmLocale\LocaleEvent;
+
+/**
+ * This class checks whether the requested uri should deliver an asset and should therefore not be redirected.
+ * If it is an asset, we return false to stop further processing of other strategies in SlmLocale\Locale\Detector.
+ *
+ * Class AssetStrategy
+ * @package SlmLocale\Strategy
+ */
+class AssetStrategy extends AbstractStrategy
+{
+    /** @var  array */
```

Comment:
maybe add description something like a list of extensions to be ignored or example

Reply:
added example in code comment and documentation.